dynamics-nav/SOAP-Web-Service-URIs.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "SOAP Web Service URIs"
+description: Describes SOAP web service URIs and contains examples.
```

Comment:
Too short per https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-seo-meta-description?branch=main.